volume-group-operator/config/rbac/role.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: manager-role
```

Comment:
can you change the `ClusterRole`, `ClusterRoleBinding`, and `ServiceAccount` names to volume-group-operator, and adjust accordingly in other related places?

Reply:
done